### PR TITLE
Fixes #RHIROS-1078 - Added new column for source_id

### PR DESCRIPTION
**Pull request by patilsuraj767**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -100,5 +100,5 @@ ifdef env
 else
 	curl -H "x-rh-identity: ${b64_identity}" \
 		 -H "x-rh-request_id: testtesttest" \
-		 http://localhost:8000/api/cost-management/v1/recommendations/openshift
-endif
\ No newline at end of file
+		 http://localhost:8000/api/cost-management/v1/recommendations/openshift | jq
```
[the remark this answers is not in the document]
Comment (patilsuraj767) [reply Makefile:103]: I don't think it is by default installed in users system.

[the remark this answers is not in the document]
Comment (r14chandra) [reply Makefile:103]: Then can we use `json.tool` here? `python -m json.tool`

[the remark this answers is not in the document]
Comment (patilsuraj767) [reply Makefile:103]: Yes, we can do that. I will make the changes.

File: internal/api/handlers.go [hunk 2]
```diff
@@ -190,6 +191,7 @@ func GetRecommendationSet(c echo.Context) error {
 		longTermSection["variation"] = variationDummyObject
 
 		recommendationSlice["id"] = recommendationSet.ID
+		recommendationSlice["source_id"] = recommendationSet.Workload.Cluster.SourceId
```
Comment (saltgen) [inline internal/api/handlers.go:194]: Do we need source_id in API responses?

**Review by saltgen — CHANGES_REQUESTED** [review]

**Review by r14chandra — APPROVED** [review]
LGTM!

File: internal/api/handlers.go (hunk 2 above)
Comment (patilsuraj767) [reply internal/api/handlers.go:194]: Not sure, I thought QE might need for verification so added it.

Comment (saltgen) [reply internal/api/handlers.go:194]: Posted a question in the Arch doc for Chris regarding the same

Comment (saltgen) [reply internal/api/handlers.go:194]: This is good to go, change approved on doc

**Review by saltgen — APPROVED** [review]

**Pull request merged**